synapse/storage/stats.py

```diff
@@ -48,6 +59,21 @@ def __init__(self, db_conn, hs):
         self.register_noop_background_update("populate_stats_process_rooms")
         self.register_noop_background_update("populate_stats_cleanup")
 
+    def quantise_stats_time(self, ts):
+        """
+        Quantises a timestamp to be a multiple of the bucket size.
+
+        Args:
+            ts: the timestamp to quantise, in seconds since the Unix Epoch
```

Comment:
```suggestion
            ts (int): the timestamp to quantise, in seconds since the Unix Epoch
```

Reply:
We also tend to keep all timestamps in the code base as milliseconds rather than seconds